sharpy/rom/utils/krylovutils.py

```diff
@@ -428,3 +479,17 @@ def remove_a12(As, n_stable):
     T2 = np.eye(n, n_stable)
     # App = T2.T.dot(T.dot(As.dot(np.linalg.inv(T).dot(T2))))
     return T, X
+
+
+def check_eye(T, Tinv, msg=''):
+    """Simple utility to verify matrix inverses"""
+    eye_approx = Tinv.dot(T)
+    max_diff = np.max(np.abs(np.eye(eye_approx.shape[0]) - eye_approx))
+
+    try:
+        log_error = np.log10(max_diff)
+        assert log_error < -6, 'T.dot(Tinv) not equal to identity, %s \nlog(error) = %.e' \
```

Comment:
Set the threshold for convergence as an optional argument in the function. If you want to deal with log of the error, please use a variable name such as `log_error` to make explicit what you are talking about.